### DISCUSSION TITLE
Getting Internal Server Error

### DISCUSSION THREAD

**Discussion by Abubakkar13** [sid86-dev/Grammarty#5] [Announcements] [answered]
Hey admin,
It's getting Internal Server error, can you check it out

**Comment by sid86-dev** [comment, marked as answer]
There is some Cloudflare bug in the module itself (Gingerit) which is probably causing this error.

**Reply by Abubakkar13** [reply]
Ok, even I tried locally same issue it's arising, like access denied

**Reply by sid86-dev** [reply]
Let's wait until this bug is fixed by the developer or I have to use any other module for the backend.

**Reply by Abubakkar13** [reply]
Yeah their is one python library that you can use, I tried, it's working as per expectation, both for grammar correction and spell check it's working.

Library link : https://pypi.org/project/language-tool-python/

Install with pip or pipe then follow it's implementation procedure. Replace ginger lit with this library.

**Reply by sid86-dev** [reply]
## Thank You so much.
### I have fixed the previous bugs and now it's working completely fine.

###  [Grammarty](https://grammarty.herokuapp.com/)

**Reply by Abubakkar13** [reply]
Great
